diff --git a/mq.c b/mq.c
index da50c2a..b3abb04 100644
--- a/mq.c
+++ b/mq.c
@@ -70,5 +70,6 @@ get_a_value(int queue_id, enum GET_SET_IDENTIFIERS field) {
         break;
 
         case SVIFP_MQ_LAST_RECEIVE_PID:
+	    printf("msg_lrpid = %d, size = %zu\n", q_info.msg_lrpid, sizeof(pid_t));
             py_value = PID_T_TO_PY(q_info.msg_lrpid);
         break;
diff --git a/tests/test_message_queues.py b/tests/test_message_queues.py
index d65fb20..70f8857 100644
--- a/tests/test_message_queues.py
+++ b/tests/test_message_queues.py
@@ -304,7 +304,7 @@ class TestMessageQueuePropertiesAndAttributes(MessageQueueTestBase):
         """exercise MessageQueue.last_receive_pid"""
         self.assertEqual(self.mq.last_receive_pid, 0)
         self.mq.send('x')
-        self.mq.receive()
+        print('recieved', self.mq.receive())
         self.assertEqual(self.mq.last_receive_pid, os.getpid())
         self.assertWriteToReadOnlyPropertyFails('last_receive_pid', 42)
